cellmlmanip/model.py

```diff
@@ -97,7 +97,7 @@ def get_free_variable(self):
             free_variable = ode.lhs.variables[0]
             return free_variable
 
-        raise ValueError('No free variable set in model.')  # pragma: no cover
+        raise ValueError('No free variable set in model.')
```

Comment:
Not sure what this exception should be. We're not really complaining about the value of a passed parameter here. Maybe a custom type?

Reply:
I was thinking of it as the value of the object you're calling it on being inappropriate for the method. So it's the `self` parameter it's complaining about? It's not really a TypeError, since the type is correct.